### PR TITLE
Recommendation around control plane sizing to handle chaotic conditions

### PR DESCRIPTION
**Pull request by chaitanyaenr**
The resource usage spikes up when one of the master in a HA cluster goes down
especially on a large and dense cluster due to unintentional conditions like
power, network, infrastructure failures or intentional conditions like cluster
shutdown to save costs. This commit adds information around the conditions for
customers and users to plan their master node requirements accordingly to avoid
cascading failures.

### CODE REVIEW

**Comment by chaitanyaenr** [comment]
@ahardin-rh @mffiedler @psuriset PTAL.

**Comment by ahardin-rh** [comment]
@chaitanyaenr Thanks for this! I'll take a look. Is there a JIRA or BZ associated with this work? Thanks!

**Comment by chaitanyaenr** [comment]
Thanks @ahardin-rh! Here is the JIRA associated with this work: https://issues.redhat.com/browse/CHAOS-19.

**Comment by ahardin-rh** [comment]
@chaitanyaenr Thanks! Should this be applied to 4.7+?

File: modules/master-node-sizing.adoc [hunk 1]
```diff
@@ -40,7 +40,7 @@ The control plane node resource requirements depend on the number of nodes in th
...
+On a large and dense cluster with three masters or control plane nodes, the CPU and memory usage will spike up when one of the nodes is stopped, rebooted or fails. The failures can be because of the unexpected issues with power, network or underlying infrastructure in addition to intentional cases where the cluster is restarted after shutting it down to save costs. The remaining two master nodes must handle the load in order to be highly available which leads to increase in the resource usage. This is also expected during upgrades because the masters are cordoned, drained, and rebooted serially to apply the operating system updates, as well as the control plane Operators update. To avoid cascading failures, keep the overall resource usage on the control plane nodes (also known as the master nodes) to at most half of all available capacity to handle the resource usage spikes. Increase the CPU and memory on the control plane nodes accordingly to avoid potential downtime due to lack of resources.
```
Comment (ahardin-rh) [inline modules/master-node-sizing.adoc:43]: rebooted, or fails. The failures can be due to unexpected issues with power, network, or underlying infrastructure

Comment (ahardin-rh) [reply modules/master-node-sizing.adoc:43]: s/master/control plane (also known as master) nodes 

We are now dropping use of the term "master" from our docs as part of the conscious language initiative.

**Comment by chaitanyaenr** [comment]
> @chaitanyaenr Thanks! Should this be applied to 4.7+?

Right.

File: modules/master-node-sizing.adoc (hunk 1 above)
Comment (chaitanyaenr) [reply modules/master-node-sizing.adoc]: Fixed it, thanks.

**Comment by ahardin-rh** [comment]
@mffiedler PTAL Thanks!

**Review by mffiedler — APPROVED** [review]
LGTM

**Pull request merged**